src/pages/Home.vue

```diff
@@ -38,11 +45,69 @@ export default {
       default: "",
       type: String,
     },
+    program: {
+      /** The program the user falls under. Eg: HaryanaStudents, DelhiStudents */
+      default: "",
+      type: String,
+    },
+    authType: {
+      /** The authentication method used by the user */
+      default: "",
+      type: String,
+    },
+  },
+  data() {
+    return {
+      textObject: {
+        placeholderText: "",
+        displayText: "",
+        addButtonText: "",
+        invalidInputMessage: "",
+        invalidLoginMessage: "",
+        submitButtonText: "",
+      },
+      maxNumberOfInput: 0,
```

Comment:
understood now after reading everything end-to-end what it means. a better name could be `maxInputsAllowed` or `maxEntriesAllowed`